projects/BFDR/NatalityRecord_submissionProperties.cs

```diff
@@ -315,7 +315,7 @@ private static bool TryParseValidDateTime(string value, out DateTimeOffset parse
                 parsedDateTime = pdt;
                 return true;
             }
-            // parseDateTime = null;
+            parsedDateTime = default;
```

Comment:
v9 won't build without `out DateTimeOffset parsedDateTime` being set to something.